src/services/services.ts

```diff
@@ -899,6 +899,13 @@ namespace ts {
         sourceFile.scriptSnapshot = scriptSnapshot;
     }
 
+    export function overrideCreateupdateLanguageServiceSourceFile(

```

Comment:
These are global functions, so I'm not sure how to avoid overriding them for all instances of the language service. (That was probably the cause of the CI failures; bad overrides are also global.)